spec/features/admin/variant_overrides_spec.rb

```diff
@@ -401,9 +401,7 @@
       let(:product) { order_cycle.products.first }
 
       before do
-        login_to_admin_section
-
-        visit 'admin/orders/new'
+        login_as_admin_and_visit 'admin/orders/new'
```

Comment:
it'd good if we could replace the hardcoded URL with its path helper while we are at it. This way we leave that up to Rails and make things slightly more changeable.

Reply:
Nevermind. I did it myself.